src/components/generic/Button/Button.scss

```diff
@@ -0,0 +1,76 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-button {
+  border-radius: 1000px;
+  cursor: pointer;
```

Comment:
isn't the browser handling the normal/disabled button cursors automatically?

Reply:
The browser does not handle the cursor in the way that we need them to do, but there would be another option:
button:not(:disabled) { cursor: pointer} which would only style the pointer for the basic state and leave the rest as default